.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
According to the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), we are required to ignore add 2 files.
`node_modules` and `test.md`, please kindly add the second file to be ignored.